practices/secrets.md

```diff
@@ -0,0 +1,92 @@
+    pgp: []
+    unencrypted_suffix: _unencrypted
+    version: 3.7.1
+```
+
+As you can see, we have access to the **key names** but the **values are encrypted**.
+Some information about which encryption key `sops` uses to encrypt/decrypt the secret is also stored under the top-level `sops` JSON key.
+In the next section we'll cover how to unencrypt this file.
+
+(secrets:sops:setup)=
+### Set up `sops`
+
+To use `sops` with a 2i2c configuration file, follow these steps:
+
+1. **Set up `sops`**. To do so, [follow the `sops` download and install instructions here](https://github.com/mozilla/sops/#1download).
+2. **Set up the Google Cloud SDK**. We use Google Cloud to provide the authentication for `sops`, and this is managed by the `gcloud` command-line tool.
+   You will need access to the `two-eye-two-see` Google project in order to access the `sops` encryption keys.
+   [Follow the Google Cloud instructions](https://cloud.google.com/sdk/docs/install) to do so.
+3. **Set up the Google Cloud Key Management Service (KMS)**. This allows you to use your Google Cloud login to provide authentication for `sops`.
+   [Follow the `sops` instructions to use KMS](https://github.com/mozilla/sops/#encrypting-using-gcp-kms).
+
+To confirm that `sops` has been set up properly, try encrypting or decrypting a configuration file per the sections below.
+
+### Decrypt secrets with `sops`
+
+In order to decrypt an encrypted configuration file, you should first follow the instructions in [](secrets:sops:setup).
+Once you've completed those steps, do the following:
+
+1. **Navigate to the root of the repository**.
+   There are a set of rules stored in [`.sops.yaml`](https://github.com/2i2c-org/pilot-hubs/blob/master/.sops.yaml) that use regex to match a file to be encrypted with the encryption key location.
+   You will receive an error from `sops` if you are not in the root folder and it cannot see this file.
+2. **Run the `sops` command**.
+   The following command will decrypt a configuration file (in this case, the configuration file in the `pilot-hubs` repository):
+
+   ```bash
+   sops --decrypt config/secrets.yaml
+   ```
+
+   :::{margin}
+   You can also use the `-d` flag instead of `--decrypt`.
+   :::
+
+   You should see the **decrypted configuration file** printed to `stdout`.
+
+   :::{admonition} Decrypt the file in-place
+   If you'd like to decrypt a configuration file in-place, use the `--in-place` or `-i` flag.
+   This will **overwrite** the configuration file with a decrypted version.
+   :::
+
+### Learn more about `sops`
+
+For more information about using `sops`, here are a few links to `sops` documentation:
+- [Basic usage of `sops`](https://github.com/mozilla/sops#usage)
+- [Encrypting files with GCP KMS](https://github.com/mozilla/sops#encrypting-using-gcp-kms)
+- [Common examples with `sops`](https://github.com/mozilla/sops#examples).a
```

Comment:
I think you have an additional `.a`, maybe you can remove it in the next PR @choldgraf 😉  ?

Reply:
oops! I fixed this up in #220